spec/requests/lead_providers/report_schools_spec.rb

```diff
@@ -4,9 +4,9 @@
 
 RSpec.describe "Report schools spec", type: :request do
   let(:user) { create(:user, :lead_provider) }
+  let!(:cohort) { create(:cohort, :current) }
 
   before do
-    create(:cohort, start_year: 2021)
```

Comment:
Is there not and advantage to doing this once, rather than before each test?

Reply:
No advantage really, those two are equivalent. I have a habit of getting all model creations out of `before` blocks, just feels cleaner but there is really no difference (expecte for the reference)